data/language/ko-KR.txt

```diff
@@ -4465,6 +4465,8 @@ STR_6155    :KDialog나 Zenity가 모두 설치되어있지 않으니, 둘 중 
 STR_6156    :이미 사용 중인 이름입니다
 STR_6157    :콘솔
 STR_6158    :이 파일을 불러오지 못했습니다...{NEWLINE}호환되지 않는 RCT 클래식 버전: {COMMA16}
+STR_6160    :검색
+STR_6161    :검색할 이름을 입력하십시오:
```

Comment:
Changes to other languages are committed to https://github.com/OpenRCT2/Localisation instead after your PR has been merged.